### ISSUE TITLE
Add "change directory" or "set directory"

### ISSUE THREAD

**Issue by MrBretze** [Tympanix/Electorrent#26]
Juste add this option to move a torrent

![Image](http://i.imgur.com/lNeMdUe.png)

**Comment by Tympanix** [comment]
Development is mostly bug fixes from now on since I am studying and doing other projects. However I will see if I can find the time. Thank you.

**Comment by FezVrasta** [comment]
Hey @tympanix I see you are maintaining again the project? Would it be possible to prioritize this? It's a very important feature :-(

**Comment by tympanix** [comment]
Yes I am maintaining the project when I have time on my hands. Unfortunately I am a bit busy at the moment, but I will make sure to prioritize this feature in the next minor release.

**Comment by darthShadow** [comment]
Any news on this?

**Comment by tympanix** [comment]
Sadly no @darthShadow. Electorrent has not had an update in quite a while. The past updates have been bug fixes and support for Deluge which means this issue was neglected yet again 😢

**Comment by darthShadow** [comment]
No problems at all. Thanks for such a good client and for supporting so many apps.

**Comment by tympanix** [comment]
Available for Transmission and qBittorrent in v2.9.0.

See 7dace3ca
